lnd.go

```diff
@@ -226,11 +228,19 @@ func Main(lisCfg ListenerCfg) error {
 	ltndLog.Infof("Opening the main database, this might take a few " +
 		"minutes...")
 
+	cfg.DB.Path = graphDir // TODO(bhandras)
+	cfg.DB.Name = "channel.db"
+	chanDbBackend, err := cfg.DB.GetBackend()
```

Comment:
Here's what I mean that `OpenWithBackend` assumes the backend is already obtained. The method is more like create/init with backend.

Reply:
Yeah, I agree fully. Renamed `OpenWithBackend` to `CreateWithBackend`.